src/res_core.ml

```diff
@@ -5411,7 +5411,7 @@ and parseJsImport ~startPos ~attrs p =
       let decl =
         match parseJsFfiDeclaration p with
         | Some decl -> decl
-        | None -> assert false
+        | None -> JsFfi.emptyDecl p.startPos
```

Comment:
See how other errors are handled.
E.g. just a few lines above here:
```ml
Parser.err ~startPos:attrLoc.loc_start ~endPos:attrLoc.loc_end p
        (Diagnostics.message (ErrorMessages.attributeWithoutNode attr));
```

Currently it goes all the way to the end of the file silently (e.g. in the provided test).

Reply:
Also see this example for where to define the default value to use (inside `Recover`:
```ml
    Parser.err p (Diagnostics.unexpected token p.breadcrumbs);
    Recover.defaultModuleExpr ()

```

Reply:
Please check what I added for error handling.
And, do you mean it's better to define the default value(`JsFfi.emptyDecl`, I made) inside `Recover` module?
It seems that all default values in `Recover` are Ast of Ocaml compiler, while Js ffi decls are not.